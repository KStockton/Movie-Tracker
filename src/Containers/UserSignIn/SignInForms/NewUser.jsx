import React from "react";
export default function NewUser(props) {
  return (
    <section className="user-sign-in-container">
      <form
        onSubmit={props.handleLogin}
        className="create-account-form login-form"
      >
        <label>Name</label>
        <input
          name="name"
          type="text"
          onChange={props.handleChange}
          value={props.name}
        />
        <label>Email</label>
        <input
          name="email"
          type="email"
          onChange={props.handleChange}
          value={props.email}
        />
        <label>Password</label>
        <input
          name="password"
          type="password"
          onChange={props.handleChange}
          value={props.password}
        />
        <button type="submit">Create Account</button>
        <button onClick={props.handleToggleForm}>
          Already have an account? Sign in!
        </button>
      </form>
    </section>
  );
}
