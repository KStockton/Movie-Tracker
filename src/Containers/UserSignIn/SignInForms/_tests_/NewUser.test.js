import React from "react";
import NewUser from '../NewUser'
import { shallow } from "enzyme";
describe("NewUser", () => {
    let wrapper;
    beforeEach(() => {
      wrapper = shallow(<NewUser />);
    });
  
     it("NewUser component should match the snapshot", () => {
      expect(wrapper).toMatchSnapshot();
    })
})