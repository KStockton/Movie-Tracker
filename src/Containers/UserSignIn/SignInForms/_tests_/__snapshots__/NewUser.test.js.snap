// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewUser NewUser component should match the snapshot 1`] = `
<section
  className="user-sign-in-container"
>
  <div
    className="user-sign-in-background"
  >
    <img
      alt="movies"
      src="https://i.pinimg.com/originals/86/f1/99/86f199d0c6f5cedd14c121164fa5fafc.jpg"
    />
  </div>
  <form
    className="create-account-form login-form"
  >
    <h3
      className="user-form-title"
    >
      NEW USER
    </h3>
    <div
      className="input-container"
    >
      <input
        className="user-input"
        maxLength="24"
        minLength="1"
        name="name"
        placeholder="Name..."
        type="text"
      />
    </div>
    <div
      className="input-container"
    >
      <input
        className="user-input"
        maxLength="24"
        minLength="2"
        name="email"
        placeholder="Email..."
        type="email"
      />
    </div>
    <div
      className="input-container"
    >
      <input
        className="user-input"
        maxLength="12"
        minLength="8"
        name="password"
        placeholder="Password..."
        type="password"
      />
    </div>
    <span
      className="status-msg"
    />
    <div
      className="button-container"
    >
      <button
        className="user-form-button sign-in-button"
        type="submit "
      >
        Create Account
      </button>
      <button
        className="user-form-link"
      >
        Already have an account? Sign in!
      </button>
    </div>
  </form>
</section>
`;
